packages/adapter-vercel/index.js

```diff
@@ -444,7 +444,15 @@ function static_vercel_config(builder, config, dir) {
 	/** @type {import('./index.js').ImagesConfig | undefined} */
 	const images = config.images;
 
-	for (const [src, redirect] of builder.prerendered.redirects) {
+	for (let [src, redirect] of builder.prerendered.redirects) {
+		if (src.replace(/\/$/, '') !== redirect.location.replace(/\/$/, '')) {
+			// handle the extreme edge case of a `/foo` -> `/foo/` redirect,
```

Comment:
I'm on mobile, so maybe I'm just misreading, but I feel like this comment should apply to the `if` statement rather than this line

This line could probably use a different comment about disregarding the presence/absence of trailing slash in prerendered redirect

Reply:
yeah the position of the comment is tricky because even if it's outside the `if` it looks like it's describing what happens inside rather than when you don't get there

fixed slightly unconventionally: inverted the condition and made the antecedent a no-op, just the comment

Reply:
lol. that works

I probably would have done it by changing the wording to something like "only apply if we're not hitting the extreme edge case of..."

but I guess I'm happy with this too. maybe the bundler will make them the same anyway :laughing: